src/styles/index.scss

```
@@ -1,3 +1,125 @@
+
+@import "./utils/variables\ ";
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+width: 198px;
+height: 406px;
+background-color: $color-white;
+border: 1px solid #f3f3f3;
+border-radius: 5px;
+margin: 20px 48px;
```

```suggestion
.card {
  width: 198px;
  height: 406px;
  background-color: $color-white;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
  margin: 20px 48px;
```

All properties inside the rule block should have an indentation.